src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureArea {
```

Comment:
The name 'FigureArea' for the interface suggests an action rather than a capability or characteristic, which is not recommended for naming interfaces. Consider renaming the interface to something like 'AreaCalculable' to better convey the purpose of the interface.